src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublishingService.cs

```diff
@@ -1,26 +1,25 @@
 #nullable enable
 using System;
+using System.Threading;
 using System.Threading.Tasks;
 using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
 using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces.Security;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
 using GovUk.Education.ExploreEducationStatistics.Common.Model;
-using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
 using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces.Security;
-using GovUk.Education.ExploreEducationStatistics.Common.Utils;
 using GovUk.Education.ExploreEducationStatistics.Content.Model;
 using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
 using GovUk.Education.ExploreEducationStatistics.Publisher.Model;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 using static GovUk.Education.ExploreEducationStatistics.Admin.Validators.ValidationErrorMessages;
 using static GovUk.Education.ExploreEducationStatistics.Admin.Validators.ValidationUtils;
-using static GovUk.Education.ExploreEducationStatistics.Publisher.Model.PublisherQueues;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Services
 {
     public class PublishingService(
-        IPersistenceHelper<ContentDbContext> persistenceHelper,
```

Comment:
Out of interest, do we still need the PersistenceHelper at all now or can it be removed?

To be honest, I'm not sure the history of it and why it was added in the first place. I assume to limit the number of db connections?

Reply:
I think it was only ever intended to be a helper to provide `CheckEntityExists` but I think that can be replaced in every case now with the methods such as `FirstOrNotFoundAsync` now. See https://github.com/dfe-analytical-services/explore-education-statistics/blob/290cb4b7b2548641bde87b76031b79042899b813/src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/QueryableExtensions.cs#L13 now.